extensions/ipynb/package.json

```diff
@@ -6,15 +6,16 @@
 	"version": "1.0.0",
 	"license": "MIT",
 	"engines": {
-		"vscode": "^1.57.0"
+		"vscode": "^1.57.0",
+		"azdata": "*"
 	},
 	"enabledApiProposals": [
 		"documentPaste",
 		"diffContentOptions",
 		"dropMetadata"
 	],
 	"activationEvents": [
-		"*"
+		"onNotebook:jupyter-notebook"
```

Comment:
Is this just a perf improvement you're doing?

Reply:
Yeah, it's so that we don't activate the extension if it's not necessary. This also reverts the line to be the same as the vscode version of this file, which makes merges a little easier